samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md

```diff
@@ -67,9 +67,18 @@ def np2csv(arr):
     return csv.getvalue().decode().rstrip()
 
 # Prepare input for the model
-urllib.request.urlretrieve("http://deeplearning.net/data/mnist/mnist.pkl.gz", "mnist.pkl.gz")
-with gzip.open('mnist.pkl.gz', 'rb') as f:
-    train_set, _, _ = pickle.load(f, encoding='latin1')
+# Load the dataset
+s3 = boto3.client("s3")
+data_bucket = S3DataConfig(
+    sm_session, "example-notebooks-data-config", "config/data_config.json"
+).get_data_bucket()
+print(f"Using data from {data_bucket}")
+
+s3.download_file(
+    data_bucket, "datasets/image/MNIST/mnist.pkl.gz", "mnist.pkl.gz")
```

Comment:
how big is this file ? 
make sure it is not too big for example script

Reply:
the input data is around ~28mb. The test_data and train_data are each around 150-180mb which is similiar to that of the sample data used in our ack sagemaker repo